### PR TITLE
Added iosevka

### PR DESCRIPTION
**Pull request by chuchana**
http://be5invis.github.io/Iosevka/

### CODE REVIEW

File: Casks/font-iosevka.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask :v1 => 'font-iosevka' do
+  version '1.0.0'
+  sha256 '66df1ac32e0e702c9666d30112db139be3d10da042c99c1c701d1e8f27121d22'
+
+  url 'https://github.com/be5invis/Iosevka/releases/download/v1.0.0/iosevka-1.0.0.tar.bz2'
```
Comment (vitorgalvao) [inline Casks/font-iosevka.rb]: Please add an `appcast 'https://github.com/be5invis/Iosevka/releases.atom'` line under `url`.

**Comment by chuchana** [comment]
done

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 0f36335b1654225a7f3efff5c407f6739712a41f. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by chuchana** [comment]
Thank you for the credit, that wasn't necessary. I don't know why I could neither squash the commits with magit, nor directly with git, because it worked in other instances.

**Pull request closed without merging**